ui/__config__/webpack.config.js

```diff
@@ -65,7 +65,28 @@ const config = {
             react: 'preact',
         },
     },
-    resolveLoader: { modules: [path.resolve(ui, '__tools__'), 'node_modules'] },
+    resolveLoader: {
+        modules: [
+            path.resolve(
+                'node_modules',
+                '@guardian',
+                'guui',
+                'dist',
+                'lib',
+                'loaders'
+            ),
+            path.resolve(
+                '..',
+                'node_modules',
+                '@guardian',
+                'guui',
+                'dist',
+                'lib',
+                'loaders'
+            ),
+            'node_modules',
+        ],
```

Comment:
Because we don't know whether `@guardian/guui` will end up in the `ui` workspace or the root workspace 👿